remote/analyze_handler.py

```diff
@@ -94,42 +93,28 @@ def pre_analyze(analyze_id, part_number, workspace, use_cache):
 
     sources_root_path = os.path.join(analyze_dir_path, 'sources-root')
 
-    build_command_file_path = os.path.join(sources_root_path, 'build_command')
+    compile_commands_path = os.path.join(
+        sources_root_path, 'compile_commands.json')
 
-    with open(build_command_file_path) as build_action:
+    with open(compile_commands_path) as compile_commands_json:
         try:
-            build_command = build_action.readline()
+            compile_commands = json.loads(compile_commands_json.readline())[0]
```

Comment:
couldn't you use the  `json.load(compile_commands_json)` here too? Do we assume that there is only one compile command in the compile_commands_json? In that case the variable name (`compile_command` instead of `compile_commands`) and a comment would be great here.